maket1.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
```

Comment:
I cannot check how correct your styles are until they don't work on GitHub Pages. https://smogse.github.io/homework/
I'll check them when you fix all other stuff.